app-frontend/src/app/index.config.js

```diff
@@ -66,6 +66,14 @@ function config( // eslint-disable-line max-params
         };
     });
 
+    RollbarProvider.init({
+        accessToken: APP_CONFIG.rollbarClientToken,
+        captureUncaught: true,
+        payload: {
+            environment: process.env.NODE_ENV || 'production'

```

Comment:
I think we should add user information here with the authservice so that in rollbar we can identify users that might be running into errors